cpp/src/parquet/encoding_test.cc

```diff
@@ -1324,6 +1324,29 @@ class TestDeltaBitPackEncoding : public TestEncodingBase<Type> {
     CheckRoundtripSpaced(valid_bits, valid_bits_offset);
   }
 
+  void ExecuteSteps(int nvalues, int repeats, int read_batch) {
```

Comment:
Why not test different batch sizes as part of `CheckRoundtrip` instead?

Reply:
It's ok, but most test current not use it. So I'd like to keep it simple here. If later we want test `batch size` for other encoding, we can move it to `CheckRoundtrip`

Reply:
Well, it would actually be simpler without adding a separate test method... and it would exercise the code more thoroughly as well.